app/views/sephora_style_guide/home/_accordion.slim

```diff
@@ -0,0 +1,32 @@
+article
+
+  h2.h2 Accordion
```

Comment:
Hey justin they are essentially slightly different though. Collapsibles are like a component in an accordion. Collapsible can be used out of the accordion context but in this example I'm going straight for an accordion example (which is a collapsible with its header as the collapsible trigger)